fabio.properties

```diff
@@ -137,6 +137,13 @@
 # and re-issue them 24 hours before they expire. The CA for client
 # authentication is expected to be stored at secret/fabio/client-certs.
 #
+# Get vault token from file
+#
+# The 'tokenfromfile' option enables fetching the vault token from a file on the filesystem.  If
+# 'tokenfromfile' is set to "true", provide the path in the 'tokenpath' option.
```

Comment:
Capitalize Vault, please, and also update the docs for the Vault KV source a few paragraphs above, where it says "The token must be provided in the VAULT_TOKEN environment variable."

Reply:
Will do